src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsJsonEncoding.cs

```diff
@@ -65,27 +66,13 @@ void WriteSettings()
                 // End the profiles object.
                 writer.WriteEndObject();
             }
-
-            JsonSerializer? jsonSerializer = null;
             
             foreach ((string key, object value) in globalSettings)
             {
                 if (!value.IsInMemoryObject())
                 {
                     writer.WritePropertyName(key);
-                    if (value is string s)
-                        writer.WriteValue(s);
-                    if (value is int i)
-                        writer.WriteValue(i);
-                    if (value is null)
-                        writer.WriteNull();
-                    if (value is bool b)
-                        writer.WriteValue(b);
-                    else
-                    {
-                        jsonSerializer ??= JsonSerializer.Create(new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore });
-                        jsonSerializer.Serialize(writer, value);
-                    }
```

Comment:
This is one of the bugs addressed here. Because of the missing `else` keywords between these `if` blocks, for `string`, `int` and `null` values we would attempt to serialize the value twice (once in the `if` block that matched the type, and once in the `else` block at the end.)